app/controllers/hyrax/dashboard/works_controller.rb

```diff
@@ -19,6 +19,11 @@ def ensure_admin!
         def search_action_url(*args)
           hyrax.dashboard_works_url(*args)
         end
+
+        # The url of the "more" link for additional facet values
+        def search_facet_path(args = {})
+          hyrax.dashboard_works_facet_path(args[:id])
```

Comment:
Without an `id` parameter `dashboard_works_facet_path` throws `ActionController::UrlGenerationError`.  I'm not opposed to this change, but it's worth noting that are other controllers that use `args[:id]` in `search_facet_path`.